dbe/actions/actions.py

```diff
@@ -79,6 +79,11 @@ def name(self) -> Text:
 
         return "healthcheck_terms_form_dbe"
 
+    def request_next_slot(self, dispatcher, tracker, domain):
+        if not tracker.get_slot("terms"):
+            dispatcher.utter_message(template=f"utter_ask_terms_doc")
```

Comment:
I'm not too sure about this. Firstly, we're also adding `utter_ask_terms_doc` to the stories, so that will trigger it being sent. And then here, if I'm understanding this correctly, for any message, if the slot `terms` isn't set, it will send the `utter_ask_terms_doc` message, which means that it will be sent every message until that slot is set, since this method doesn't set the slot.

Reply:
In my tests I only got the doc message once despite putting it here and in the stories. but if I only put it in the stories then it doesn't seem to get sent. so I'm not sure how that works.
And yeah, this will technically send it for every slot but this form only has one slot so that is technically fine.
I guess I actually want to make sure we send it only if we're asking about the terms slot and only if terms isn't "yes". That way we'll send it even if they've previously responded with "No"

Reply:
Oh. I've just figured this out. I'll put it just in the stories then :+1:

Reply:
@rudigiesler I've opted to keep this method because it ensures that the doc is sent consistently whenever we are asking for the terms. Even if the user responds with something invalid when we re-ask the question it will send the doc again.

I haven't been able to get it to work by adding the message to the stories.